ts/api/pagopa.ts

```diff
@@ -0,0 +1,35 @@
+import { pagoPaApiUrlPrefix } from "../config";
+import { CreditCard, CreditCardId } from "../types/CreditCard";
+import { buildWalletFromCreditCard } from "../utils/converters";
+
+// Send a new version of the profile to the Proxy
+export const addCard = async (
```

Comment:
Please use the [backend client](https://github.com/teamdigitale/italia-app/blob/master/ts/api/backend.ts#L129) as a reference for implementing the PagoPA API client:

* it already has full integration with `io-ts` for validating requests and responses
* it decodes response status codes into well typed responses
* it transparently supports retries and timeouts